homework7_cows_and_bulls.py

```diff
@@ -0,0 +1,36 @@
+"""Module providing a function printing python version."""
+import random
+from typing import List
+
+print("Game 'Bulls and Cows'!")
+
+list_with_random_numbers: List[int] = []
+while len(list_with_random_numbers) != 4:
+    random_element = random.choice([1, 2, 3, 4, 5, 6, 7, 8, 9, 0])
+    if random_element not in list_with_random_numbers:
+        list_with_random_numbers.append(random_element)
+
+common_cows: List[int] = []
+common_bulls: List[int] = []
+while True:
+    common_cows.clear()
+    common_bulls.clear()
+    str_from_player = input("Enter you prediction: ")
```

Comment:
user can enter non unique number, need to handle this case

Reply:
Fixed it